Sources/Core/SourceFile.swift

```diff
@@ -26,7 +26,29 @@ public struct SourceFile {
 
   /// Creates an instance representing the file at `filePath`.
   public init(contentsOf filePath: URL) throws {
-    let storage = try Storage(filePath) { try String(contentsOf: filePath) }
+    let storage = try Storage(filePath) { try String(contentsOf: filePath)[...] }
+    self.storage = storage
+  }
+
+  /// Creates an instance for the `text` given by a multiline string literal in the given
+  /// `swiftFile`, the literal's textual content (the line after the opening quotes) being
+  /// startLine.
+  ///
+  /// N.B. The text of the instance will literally be what's in the Swift file, including its
```

Comment:
```suggestion
  /// The text of the instance will literally be what's in the Swift file, including its
```

Very minor, but let's try to avoid introducing different way to write additional notes in the docs.